sphinx/writers/html.py

```diff
@@ -115,10 +115,6 @@ def depart_desc(self, node: Element) -> None:
     def visit_desc_signature(self, node: Element) -> None:
         # the id is set automatically
         self.body.append(self.starttag(node, 'dt'))
-        # anchor for per-desc interactive data
-        if node.parent['objtype'] != 'describe' \
-           and node['ids'] and node['first']:
-            self.body.append('<!--[%s]-->' % node['ids'][0])
```

Comment:
It seems the `first` flag has been used to output this comment tag. I don't know why this is needed. Anyway, this is no longer working. So we can drop this feature now.